src/components/app-configure/AppChain.vue

```diff
@@ -1,21 +1,99 @@
+<script lang="ts">
```

Comment:
Can we use the `script setup` syntax here?

Reply:
We'll need to do changes for that. Once I'm done with Configure redesign, I'll work on migrating things to `script setup` syntax. Configure redesign will use script setup syntax